### ISSUE TITLE
dnstap_receiver can not shutdown with python3.7

### ISSUE THREAD

**Issue by motok** [dmachard/dnstap-receiver#36]
dnstap_receiver cannot shutdown itself when receiving ctl-C if it runs under python 3.7.
This is because python3.7 doesn't recognise "name" argument. It seems to be introduced in 3.8.
Simply comment-out the line 223 (name='shutdown',) and dnstap_receiver can shutdown.

- Error message
```
Exception in callback start_receiver.<locals>.<lambda>() at /usr/local/lib/python3.7/site-packages/dnstap_receiver/receiver.py:262
handle: <Handle start_receiver.<locals>.<lambda>() at /usr/local/lib/python3.7/site-packages/dnstap_receiver/receiver.py:262>
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/local/lib/python3.7/site-packages/dnstap_receiver/receiver.py", line 262, in <lambda>
  File "/usr/local/lib/python3.7/site-packages/dnstap_receiver/receiver.py", line 223, in start_shutdown_task
TypeError: create_task() got an unexpected keyword argument 'name'
```
- comment out here.
```
218 def start_shutdown_task(signal, loop, start_shutdown):
219     global shutdown_task
220     if not shutdown_task:
221         shutdown_task = asyncio.create_task(
222             shutdown(signal, loop, start_shutdown),
223             # name='shutdown',
224         )
```

**Comment by dmachard** [comment]
thanks to report that.
I had a matrix to test from python36 to 3.9 in github actions https://github.com/dmachard/dnstap-receiver/actions/runs/1007097253 and also commit the fix

**Comment by motok** [comment]
Thank you very much and I've confirmed the problem had gone.
Do you prefer close this ticket yourself or may I close it?

**Comment by dmachard** [comment]
Fix included in the new v4.4.0 release.
